### PR TITLE
Update create-dns-records-at-wix.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that wix doesn't support DNS entries for sub domain

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2238

File: microsoft-365/admin/dns/create-dns-records-at-wix.md [hunk 1]
```diff
@@ -53,6 +55,9 @@ Before you use your domain with Microsoft, we have to make sure that you own it.
   
 > [!NOTE]
 > This record is used only to verify that you own your domain; it doesn't affect anything else. You can delete it later, if you like. 
+
+> [!NOTE]
+> WIX does not support DNS entries for subdomain
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/dns/create-dns-records-at-wix.md]:
```suggestion
> WIX does not support DNS entries for subdomains.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks.

**Pull request merged**